src/index.html

```diff
@@ -7,8 +7,110 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo__header">
+      <a
+      class="header__img"
+      href="#"
+      target="_blank"
+      >
```

Comment:
```suggestion
    <header class="header">
      <a
        class="header__img"
        href="#"
      >
```
- Just `header` as children elements have classes like this `header__img`.
- Remove `target="_blank"` for all local links, you don't need to open them in a new tab. If users want to open in a new tab they can do right click => open link in a new tab.
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/1b4a795d-a187-4aea-b2ce-85aa83821f93)
- fix formatting for all elements